awx/main/tests/functional/api/test_inventory.py

```diff
@@ -471,14 +471,14 @@ def test_validating_credential_type(self, organization, inventory, admin_user, p
         assert 'Cloud-based inventory sources (such as ec2)' in r.data['credential'][0]
         assert 'require credentials for the matching cloud service' in r.data['credential'][0]
 
-    def test_vault_credential_not_allowed(self, project, inventory, vault_credential, admin_user, post):
+    def test_vault_credential_not_allowed(self, project, inventory, vault_credential, admin_user, post, compute_cloud_inventory_sources):
```

Comment:
I'm guessing you didn't get to fixing this yet, before PTO? Anyway, I'm leaving this comment to flag this place.